.travis.yml

```diff
@@ -7,6 +7,10 @@ php:
   - 5.3
```

Comment:
Build is failing with

```
PHP 5.3 is supported only on Precise.
See https://docs.travis-ci.com/user/reference/trusty#PHP-images on how to test PHP 5.3 on Precise.
Terminating.
```

See https://travis-ci.org/qandidate-labs/qandidate-toggle-api/jobs/382083705#L444-L446